packages/front-end/hooks/useStripeSubscription.ts

```diff
@@ -68,6 +68,7 @@ export default function useStripeSubscription() {
 
   return {
     freeSeats,
+    organization: data?.organization,
```

Comment:
I don't like returning the whole organization object here. I think we need to change the name of `activeAndInvitedUsers` since that's not really what it is now. This hook should return two values, the number of seats currently being paid for and the number of seats added to the org.